packages/package_info_plus/package_info_plus_windows/lib/src/file_version_info.dart

```diff
@@ -10,10 +10,10 @@ part of package_info_plus_windows;
 
 class _LANGANDCODEPAGE extends Struct {
   @Uint16()
-  int? wLanguage;
+  external int? wLanguage;
 
   @Uint16()
-  int? wCodePage;
+  external int? wCodePage;
 }
```

Comment:
This PR is mentioning CI but we have a change in the package. can you take this out and send another PR for this. Please mention what this change is going to resolve.